package.json

```diff
@@ -35,5 +34,8 @@
   "bugs": {
     "url": "https://github.com/handsomecode/trello-parent-child/issues"
   },
-  "homepage": "http://labs.handsome.is/trello/"
+  "homepage": "http://labs.handsome.is/trello/",
+  "dependencies": {
```

Comment:
This also appears to be outside the scope of the actual work on this feature, so normally we'd put it on its own `feature/add-gulp` branch. Don't sweat it this time around, though.

Reply:
This must have been one of the build tools or something similar. I don't actually recall making this change myself.